synfig-studio/src/gui/widgets/widget_timetrack.h

```diff
@@ -95,6 +97,7 @@ class Widget_Timetrack : public Widget_TimeGraphBase
 	ActionState get_action_state() const;
 	void set_action_state(ActionState action_state);
 	sigc::signal<void>& signal_action_state_changed() { return signal_action_state_changed_; }
+	bool waypoint_mouse_copy=false;
```

Comment:
1. This shouldn't be public
2. Use spaces surrounding binary operators

Reply:
regarding one - hmm, I have always wondered why its always better to keep the members private. I mean I know its for "encapsulation" but for example here I set this flag from outside the class, so if this is private I'd have to use asetter ,which would be creating a whole new function that would do something which would be equivalent to just accessing right away and setting.